### PR TITLE
Editorial: avoid "The Type of"

### PR DESCRIPTION
**Pull request by ljharb**

### CODE REVIEW

**Review by michaelficarra — APPROVED** [review]

File: spec.html [hunk 1]
```diff
@@ -3147,7 +3147,7 @@ <h2>[[GetOwnProperty]] ( _P_ )</h2>
             The normal return type is either Property Descriptor or Undefined.
           </li>
           <li>
-            If the Type of the return value is Property Descriptor, the return value must be a fully populated Property Descriptor.
+            If the return value is a Property Descriptor, the return value must be a fully populated Property Descriptor.
```
Comment (jmdyck) [inline spec.html:3150]:
```suggestion
            If the return value is a Property Descriptor, it must be a fully populated Property Descriptor.
```
(reads a bit better)

File: spec.html [hunk 2]
```diff
@@ -3236,7 +3236,7 @@ <h2>[[OwnPropertyKeys]] ( )</h2>
             The returned List must not contain any duplicate entries.
           </li>
           <li>
-            The Type of each element of the returned List is either String or Symbol.
+            Each element of the returned List is a property key.
```
Comment (jmdyck) [inline spec.html:3239]:
```suggestion
            Each element of the returned List must be property key.
```
The status quo is an "is", but the neighboring points are "musts", so not sure.

File: spec.html [hunk 3]
```diff
@@ -15898,7 +15898,7 @@ <h1>[[OwnPropertyKeys]] ( ): either a normal completion containing a List of pro
             The returned List contains no duplicate entries.
           </li>
           <li>
-            The Type of each result List element is either String or Symbol.
+            Each result List element is a property key.
```
Comment (jmdyck) [inline spec.html:15901]: The status quo has "result List element", but it's an odd phrase. 
```suggestion
            Each element of the result List is a property key.
```
would be more idiomatic.

File: spec.html [hunk 4]
```diff
@@ -15898,7 +15898,7 @@ <h1>[[OwnPropertyKeys]] ( ): either a normal completion containing a List of pro
             The returned List contains no duplicate entries.
           </li>
           <li>
-            The Type of each result List element is either String or Symbol.
+            Each element of the result List element is a property key.
```
Comment (michaelficarra) [inline spec.html:15901]:
```suggestion
            Each element of the returned List is a property key.
```

For symmetry with above step (plus grammar error).

File: spec.html (hunk 2 above)
Comment (michaelficarra) [reply spec.html:3239]: This is true, but on 15901 we have the exact same sentence. So maybe the other "musts" should change? Or maybe it doesn't matter that we're consistent with them.

Comment (ljharb) [reply spec.html:3239]: I changed this one, happy to change them back if you want

File: spec.html (hunk 4 above)
Comment (jmdyck) [reply spec.html:15901]: (Yeah, but on the other hand, the subsequent points refer to "the result List".)

**Review by syg — APPROVED** [review]

**Pull request merged**